cmd/keyaliases/crtka.go

```diff
@@ -93,7 +93,7 @@ func init() {
 	CreateCmd.Flags().StringVarP(&name, "alias", "s",
 		"", "Name of the key alias")
 	CreateCmd.Flags().StringVarP(&format, "format", "f",
-		"", "Format of the certificate; selfsignedcert, pem or pkcs12 (file extn is .pfx)")
+		"", "Format of the certificate; selfsignedcert, keycertfile or pkcs12 (file extn is .pfx)")
```

Comment:
change to `Format of the certificate; selfsignedcert, keycertfile (pem) or pkcs12 (file extn is .pfx)`